packages/xpetra/src/Vector/Xpetra_TpetraVector_def.hpp

```diff
@@ -115,11 +115,11 @@ void TpetraVector<Scalar,LocalOrdinal,GlobalOrdinal,Node>::describe(Teuchos::Fan
 { XPETRA_MONITOR("TpetraVector<Scalar,LocalOrdinal,GlobalOrdinal,Node>::describe"); getTpetra_Vector()->describe(out, verbLevel); }
 
 template<class Scalar, class LocalOrdinal, class GlobalOrdinal, class Node>
-Scalar TpetraVector<Scalar,LocalOrdinal,GlobalOrdinal,Node>::dot(const Vector &a) const
+Scalar TpetraVector<Scalar,LocalOrdinal,GlobalOrdinal,Node>::dot(const Vector<Scalar,LocalOrdinal, GlobalOrdinal, Node > &a) const
 { XPETRA_MONITOR("TpetraVector<Scalar,LocalOrdinal,GlobalOrdinal,Node>::dot"); return getTpetra_Vector()->dot(*toTpetra(a)); }
 
 template<class Scalar, class LocalOrdinal, class GlobalOrdinal, class Node>
-typename Teuchos::ScalarTraits< Scalar >::magnitudeType TpetraVector<Scalar,LocalOrdinal,GlobalOrdinal,Node>::normWeighted(const Vector &weights) const
+typename Teuchos::ScalarTraits< Scalar >::magnitudeType TpetraVector<Scalar,LocalOrdinal,GlobalOrdinal,Node>::normWeighted(const Vector<Scalar,LocalOrdinal, GlobalOrdinal, Node > &weights) const
```

Comment:
Tpetra has deprecated this.  Does MueLu actually need it still?  (Would be good to know that before Tpetra removes it entirely ;-) .)

Reply:
Grep says we don't use it.

Reply:
@csiefer2 I'm not sure anyone would consider most of Xpetra a public interface, so I think it's fair game to purge or at least deprecate.

Reply:
I'm totally OK with purging it when Tpetra does.